warehouse/integrations/mssql/mssql.go

```diff
@@ -57,28 +57,43 @@ var rudderDataTypesMapToMssql = map[string]string{
 }
 
 var mssqlDataTypesMapToRudder = map[string]string{
-	"integer":                  "int",
-	"smallint":                 "int",
-	"bigint":                   "int",
-	"tinyint":                  "int",
-	"double precision":         "float",
-	"numeric":                  "float",
-	"decimal":                  "float",
-	"real":                     "float",
-	"float":                    "float",
-	"text":                     "string",
-	"varchar":                  "string",
-	"nvarchar":                 "string",
-	"ntext":                    "string",
-	"nchar":                    "string",
-	"char":                     "string",
-	"datetimeoffset":           "datetime",
+	"bigint":   "int",
+	"int":      "int",
+	"smallint": "int",
+	"tinyint":  "int",
+	"integer":  "int",
+
+	"numeric":          "float",
+	"decimal":          "float",
+	"money":            "float",
+	"small money":      "float",
+	"real":             "float",
+	"float":            "float",
+	"double precision": "float",
+
+	"char":             "string",
+	"nchar":            "string",
+	"varchar":          "string",
+	"nvarchar":         "string",
+	"text":             "string",
+	"ntext":            "string",
+	"binary":           "string",
+	"image":            "string",
+	"varbinary":        "string",
+	"uniqueidentifier": "string",
+
 	"date":                     "datetime",
+	"datetimeoffset":           "datetime",
 	"datetime2":                "datetime",
+	"smalldatetime":            "datetime",
+	"datetime":                 "datetime",
+	"time":                     "datetime",
 	"timestamp with time zone": "datetime",
 	"timestamp":                "datetime",
-	"jsonb":                    "json",
-	"bit":                      "boolean",
+
+	"bit": "boolean",
+
+	"jsonb": "json",
```

Comment:
These datatypes we have added: 
int, money, small money, binary, image, varbinary, uniqueidentifier